### PR TITLE
[docs] Add default TTL value

### LINKED ISSUE #9415: What's the default value for TTL
**Describe the bug**
When I check `set-message-ttl` in Pulsar docs, and found that the default values for it changes since Pulsar 2.6.2.

image

Is the default value `0` before 2.6.2 and `3600` after 2.6.2, or does one default value (`0` or `3600`) apply for all versions?

### PR DESCRIPTION
**Pull request by Jennifer88huang**
Fixes #9415 

### Motivation
It's not clear to users whether setting `ttlDurationDefaultInSeconds=0` means disabling TTL.

### Modifications
Make the default setting and values much clearer

### CODE REVIEW

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ internalListenerName|Specify the internal listener name for the broker.<br><br>*
 |brokerClientTlsTrustStorePassword| TLS TrustStore password for internal client, used by the internal client to authenticate with Pulsar brokers ||
 |brokerClientTlsCiphers| Specify the tls cipher the internal client will use to negotiate during TLS Handshake. (a comma-separated list of ciphers) e.g.  [TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]||
 |brokerClientTlsProtocols|Specify the tls protocols the broker will use to negotiate during TLS handshake. (a comma-separated list of protocol names). e.g.  [TLSv1.2, TLSv1.1, TLSv1] ||
-|ttlDurationDefaultInSeconds|  The default ttl for namespaces if ttl is not configured at namespace policies.  |0|
+|ttlDurationDefaultInSeconds| The default TTL for namespaces if TTL is not configured at namespace policies. TTL(time to live) is disabled by default and the value is set to `0`. |0|
```
Comment (sijie) [inline site2/docs/reference-configuration.md]: @Jennifer88huang should we call out that setting it to `0` is disabling TTL and the default value is `0`.

Also, shall we update the documentation of other releases?

Comment (Jennifer88huang) [reply site2/docs/reference-configuration.md]: @sijie you're so fast, I've just updated it to be more clear. Please check the [latest version above](https://github.com/apache/pulsar/pull/9395/commits/4e04618586577841c1dd258b35d0ffa1182712cf)

Comment (Jennifer88huang) [reply site2/docs/reference-configuration.md]: Yes, we will apply changes to other versions if it should. 
Usually, we update in the master and finalize the change first, and then apply the final version content to other versions. Thank you for your reminder.

**Comment by Anonymitaet** [comment]
Need to update the [admin doc](http://pulsar.apache.org/tools/pulsar-admin/2.7.0-SNAPSHOT/#-em-set-message-ttl-em-) as well?

**Review by eolivelli — APPROVED** [review]

File: site2/docs/reference-configuration.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ internalListenerName|Specify the internal listener name for the broker.<br><br>*
 |brokerClientTlsTrustStorePassword| TLS TrustStore password for internal client, used by the internal client to authenticate with Pulsar brokers ||
 |brokerClientTlsCiphers| Specify the tls cipher the internal client will use to negotiate during TLS Handshake. (a comma-separated list of ciphers) e.g.  [TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]||
 |brokerClientTlsProtocols|Specify the tls protocols the broker will use to negotiate during TLS handshake. (a comma-separated list of protocol names). e.g.  [TLSv1.2, TLSv1.1, TLSv1] ||
-|ttlDurationDefaultInSeconds|  The default ttl for namespaces if ttl is not configured at namespace policies.  |0|
+|ttlDurationDefaultInSeconds|The default TTL(time to live) for namespaces if TTL is not configured at namespace policies. When the value is set to `0`, TTL is disabled. It is disabled by default. |0|
```
Comment (Huanli-Meng) [inline site2/docs/reference-configuration.md:218]:
```suggestion
|ttlDurationDefaultInSeconds|The default Time to Live (TTL) for namespaces if the TTL is not configured at namespace policies. When the value is set to `0`, TTL is disabled. By default, TTL is disabled. |0|
```

Comment (Jennifer88huang) [reply site2/docs/reference-configuration.md]: @Huanli-Meng Thank you for your suggestion, updated, PTAL again.

**Comment by Jennifer88huang** [comment]
> Need to update the [admin doc](http://pulsar.apache.org/tools/pulsar-admin/2.7.0-SNAPSHOT/#-em-set-message-ttl-em-) as well?

Thanks for your reminder. Is this written manually or generated automatically?

**Comment by Anonymitaet** [comment]
> > Need to update the [admin doc](http://pulsar.apache.org/tools/pulsar-admin/2.7.0-SNAPSHOT/#-em-set-message-ttl-em-) as well?
> 
> Thanks for your reminder. Is this written manually or generated automatically?

Descriptions are added manually.

**Review by Anonymitaet — APPROVED** [review]

**Review by Huanli-Meng — APPROVED** [review]

**Comment by Jennifer88huang** [comment]
Confirmed with eng, the default value for `set-message-ttl` and `ttlDurationDefaultInSeconds` is `0`.  Update the default value and description for different versions.

**Comment by Jennifer88huang** [comment]
@sijie @codelipenghui PTAL again, thank you.

**Review by sijie — APPROVED** [review]

**Pull request merged**